packages/account/src/Styles/account.scss

```diff
@@ -768,7 +769,7 @@ $MIN_HEIGHT_FLOATING: calc(
 
         &-leave-confirm,
         &-submit-success {
-            @include desktop-screen {
+            @include desktop {
```

Comment:
why did we change this ?
This should be "desktop-screen" as per new breakpoint. isnt?